### PR TITLE
Resolve comments for crd upgrade verify

### PR DESCRIPTION
**Pull request by qiujian16**

### CODE REVIEW

**Comment by deads2k** [comment]
/lgtm
/approve
/hold

just holding to remove the forced failure that we see evidence of in https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_api/895/pull-ci-openshift-api-master-verify/1382343659604676608

**Comment by qiujian16** [comment]
@deads2k I did some more update

1. prow only has master branch, so the script find the v1beta1  crd from 1 commit before instead of switching branch.
2. update some other parts and use clusterversionoperator to test...

File: hack/verify-crds-version-upgrade.sh [hunk 1]
```diff
@@ -38,21 +43,21 @@ apiserverstart
 # sleep 5 second to wait for apiserver start
 sleep 5
 
-CURRENT_BRANCH=$(git branch --show-current)
+CURRENT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+LAST_COMMIT=$(git rev-parse HEAD~1)
 # Switch to master branch and apply all the existing v1beta1 crds
-git checkout master
```
Comment (deads2k) [inline hack/verify-crds-version-upgrade.sh:43]: huh.  I thought master was present

**Comment by deads2k** [comment]
/lgtm


Thanks for the test.  pull it out and this can merge.

**Comment by deads2k** [comment]
adding valid bug. this is a verify script, not shipped product code.

**Pull request merged**